requirements.txt

```diff
@@ -6,9 +6,9 @@ Flask-Wtf>=1.1 # BSD-3-Clause
 authlib>=1.2 # BSD-3-Clause
 # Requests is an unlisted dependency of authlib
 requests>=2.28 # Apache-2.0
-pyyaml>=6.0 # MIT
+pyyaml>=6.0.1 # MIT
```

Comment:
you are not using in this change anything requiring explicitly 6.0.1 neither it is fixing critical vulnerabilities. Due to the fact, that it was ">=6.0" every time the project is being installed it fetches latest available version, so you do not need to change it. But it also doesn't harm